### PR TITLE
AP-567 set up uswitch_events_v2 and config for forwarder dev

### PR DESCRIPTION
**Pull request by dan-dawson**

### CODE REVIEW

File: dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf [hunk 1]
```diff
@@ -38,6 +38,35 @@ resource "kafka_topic" "uswitch_events_v1" {
...
+    # keep data locally for 1 hour
+    "local.retention.ms" = "3600000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+# tflint-ignore: terraform_naming_convention
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:61]: Please fix this name

Comment (dan-dawson) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:61]: I can do if it's really necessary. We have used the same naming throughout this file, and we borrowed the structure of this from another namespaces files...

I'd rather not change it and have one odd module if possible

Comment (dan-dawson) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf]: Done 👍🏻

File: dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf [hunk 2]
```diff
@@ -38,6 +38,35 @@ resource "kafka_topic" "uswitch_events_v1" {
   }
 }
 
+resource "kafka_topic" "uswitch_events_v2" {
+  name = "customer-proposition.uswitch.events.v2"
+
+  replication_factor = 3
+  partitions         = 15
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:45]: Is this a migrated topic? If not, can it do with less partitions?

Comment (dan-dawson) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:45]: Potentially. We are migrating from v1 to v2, but do plan to cut down the number of events in v2. 
Do you have a suggestion on how many partitions is preferred?

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:45]: It boils down to how many parallel consumers you need, depending on the throughput needs of your consumers.
If you're uncertain, just go with 5, as you can scale it up afterwards

Comment (dan-dawson) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf]: We can give 5 a go. That's done now 👍🏻

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**